pkg/apis/sriovnetwork/v1/sriovnetworknodepolicy_types.go

```diff
@@ -21,6 +21,9 @@ type SriovNetworkNodePolicySpec struct {
 	NicSelector  SriovNetworkNicSelector `json:"nicSelector"`
 	DeviceType   string                  `json:"deviceType,omitempty"`
 	IsRdma       bool                    `json:"isRdma,omitempty"`
+	TotalVfs     int                     `json:"totalVfs,omitempty"`
+	LinkType     string                  `json:"linkType,omitempty"` 
+	EnableSriov  bool                    `json:"enableSriov,omitempty"`
```

Comment:
I don't think we need to add `EnableSriov` field explicitly here. When a user specified `NumVfs`, it implies that SR-IOV shall be enable.